content/en/index.md

```diff
@@ -7,7 +7,7 @@ description: 'Fast 3kB alternative to React with the same modern API.'
 
 <jumbotron>
     <h1>
-        <logo height="1.5em" title="Preact" text inverted>Preact</logo>
+        <logo height="1.5em" title="Preact" text="true" inverted="true">Preact</logo>
```

Comment:
Breaking change in `preact-markup`: https://github.com/developit/preact-markup/pull/42#issuecomment-1922266537